python/cuml/dask/ensemble/randomforestregressor.py

```diff
@@ -127,32 +132,47 @@ def __init__(
         client=None,
         verbose=False,
         n_estimators=10,
+        random_state=None,
         seed=None,
         ignore_empty_partitions=False,
         **kwargs
     ):
         super(RandomForestRegressor, self).__init__(client=client,
                                                     verbose=verbose,
                                                     **kwargs)
+
+        if seed is not None:
+            if random_state is None:
+                warnings.warn("Parameter 'seed' is deprecated and will be"
+                              " removed in 0.17. Please use 'random_state'"
+                              " instead.",
+                              DeprecationWarning)
+                random_state = seed
+            else:
+                warnings.warn("Both 'seed' and 'random_state' parameters were"
+                              " set. Using 'random_state' since 'seed' is"
+                              " deprecated and will be removed in 0.17.",
+                              DeprecationWarning)
```

Comment:
Since this logic actually is in randomforest_common.pyx, is it necessary to have it in both the classifier and regressor?

Reply:
Yes, thanks for catching this. I did that in the classifier but looks like I forgot to update it for the regressor.

Reply:
I think it's easiest if we keep this as is for this release, unfortuantely.

I think we can rely on the base single GPU class if we pass both seed and random_state down all the way down call stack, but it gets a bit messy with the `_create_model` and kwargs due to already having some arguments explicitly spelled out and passed through. Doable, but I think it adds more complexity and may not be worth it.

@dantegd do you think it's worth it? If so, happy to implement it.

Reply:
nah, I think it is fine as it is since as you say it will be gone in a version. I thought it was just a matter of deleting, but not worth more time investment than that

Reply:
👍